runtime/src/bank.rs

```diff
@@ -35,6 +35,7 @@
 //! already been signed and verified.
 #[allow(deprecated)]
 use solana_sdk::recent_blockhashes_account;
+pub use solana_sdk::reward_type::RewardType;
```

Comment:
nit: what do you think about marking this re-export as deprecated? That'll just require changing the other places that import this

Reply:
Ah, that's a nice idea. The other monorepo importers can just be updated to point to sdk.

Reply:
Sadly, it turns out a deprecation tag here won't actually raise a warning for users: https://github.com/rust-lang/rust/issues/30827

Reply:
Ah, that's too bad.  Thanks for looking into it